spec/furniture/marketplace/collecting_payments_system_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+# @see https://github.com/zinc-collective/convene/issues/1576
```

Comment:
Why are these back references here? I was assuming it might point to a specific comment or something in the issue related to the test. New documentation pattern?

Reply:
So, the way I am using issues is a little different than most teams; where all issues are short lived. Instead, I'm using long-lived issues to describe the overall feature definition. The backlink brings you to the long-lived issue describing the "`Marketplace`: Collecting Payments" value proposition.

The issue itself then links out to whatever other evergreen/long-lived documentation we want; like product marketing copy, interaction design wireframes, etc. 

If this test gets to be too big, we could sprout a `marketplace/collecting_payments/via_stripe_system_spec.rb` which would link to the issue  that describes that particular dip down into the fractal of super-powers we're providing.

Reply:
👍🏻 Cool. Yeah, I get that we work with issues as a combination of living spec. So basically you're just pointing back to design docs. Which I think makes sense. I guess I was wondering if you would, then, point back to issues on all unit tests, or just feature specs, etc....

Reply:
I am planning to map system specs back to the issue definitions; since trying to do it on a class-structure level feels liek a bit too tight of coupling to keep consistent; whereas the system specs should be exercising a single use-case.